src/app.module.ts

```diff
@@ -95,8 +95,8 @@ export const REST_MODULES = [
         IRONFISH_API_KEY: joi.string().required(),
         IRONFISH_PHASE_ONE_API_URL: joi.string().required(),
         IRONFISH_PHASE_TWO_API_URL: joi.string().required(),
-        JUMIO_API_SECRET: joi.string().required(),
-        JUMIO_API_TOKEN: joi.string().required(),
+        JUMIO_API_SECRET: joi.string().optional().allow(''),
```

Comment:
I think it'd be weird to spin up the API without the secret / token since we'll need those values to function. We use `test` (ex. `MAGIC_SECRET_KEY=test` in `.env.template`) and then that value can be updated in `.env` if needed. I'd vote to keep this as `required()` but use `test` or something in the template. What do you think?

Reply:
ok, that's fine by me. Basically, this is where you would use a staging/dummy DB value, but I don't think we have that. The template value doesn't work so wanted to fixt aht.